packages/nuejs.org/site.yaml

```diff
@@ -6,7 +6,7 @@ libs: ["@lib", docs/lib, blog/lib]
 og: /img/og-blue-big.png
 view_transitions: true
 native_css_nesting: true
-origin: https://www2.nuejs.org
+origin: https://www2.nuejs.org # TODO: update to correct host on release
```

Comment:
I can also already change it, but I don't know when you would move to the main domain.

Reply:
Good point. I'll change that tomorrow.